LibraryTests/QuantumPhaseEstimationTests.qs

```diff
@@ -0,0 +1,54 @@
+namespace Microsoft.Quantum.Tests {
+
+    open Microsoft.Quantum.Primitive;
+    open Microsoft.Quantum.Canon;
+	open Microsoft.Quantum.Extensions.Convert;
```

Comment:
It looks like you have a mix of spaces and tabs here, would you be willing to normalize to four spaces?